cds_books/ldap/admin.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# CDS Books is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+"""Admin panel for ldap synchronizations."""
+
+import flask
+from flask import Blueprint, flash, redirect, request
+from flask_admin.base import expose
+from flask_admin.contrib.sqla import ModelView
+from flask_admin.helpers import get_redirect_target
+from flask_babelex import gettext as _
+from invenio_db import db
+
+from cds_books.ldap.models import LdapSynchronizationLog
+from cds_books.ldap.tasks import synchronize_users_task
+
+
+class LdapSynchronizationLogModelView(ModelView):
+    """Invenio admin view for ldap users synchronization."""
+
+    can_create = True
```

Comment:
how are these attributes used?

Reply:
Those come from `flask-admin`, which is itself provided by `invenio-admin` (see [the documentation](https://flask-admin.readthedocs.io/en/latest/introduction#modelview-configuration-attributes)).